### ISSUE TITLE
Herbstluftwm - transparent windows

### ISSUE THREAD

**Issue by AndrewOlsen** [naelstrof/maim#115]
I noticed while using Herbstluftwm that maim thinks windows are transparent if the attribute frame_bg_transparent is set to true.

Here is a screen of the frames with a transparent background:
![transparent_true](https://user-images.githubusercontent.com/13722163/30983347-24d7e934-a458-11e7-850f-893a723d0ff1.png)

And here is the same screen with non transparent background:
![transparent_false](https://user-images.githubusercontent.com/13722163/30983359-2e513fce-a458-11e7-9867-5b54329ba90f.png)

**Comment by naelstrof** [comment]
Which main version are you using?

**Comment by AndrewOlsen** [comment]
v5.4.66

**Comment by naelstrof** [comment]
Try v5.4.68 please.

**Comment by AndrewOlsen** [comment]
That version is working correctly

**Comment by naelstrof** [comment]
Just to make sure, when you click on the transparent terminals while running `maim -s`, does it work as you intend? (It should screenshot the transparent window by itself, without any background).

To get around this functionality, you can run `maim -g $(slop)` instead.

**Comment by AndrewOlsen** [comment]
`maim -s` is also working as intended.

Screenshot:
![test](https://user-images.githubusercontent.com/13722163/30994034-cc442a12-a47f-11e7-9a2d-e18ca2ce4c60.png)

**Comment by naelstrof** [comment]
`maim -s`should still have the transparency of the terminal, the image generated should be the transparent terminal composed onto a transparent background. It looks to me like the transparency was discarded in that image.

Could you give me a full screen shot with terminal transparency enabled(`maim`), with a shot of the specific terminal with transparency enabled(`maim -s`)?

**Comment by AndrewOlsen** [comment]
I apologize that I wasn't clear enough, my terminals have no transparency. A feature of the window manager allows the frame to have a transparent background like so:

![frame](https://user-images.githubusercontent.com/13722163/30994529-6ebd36e2-a482-11e7-874c-020566f998b0.png)

instead of like this:
![solid_frame](https://user-images.githubusercontent.com/13722163/30994534-7923df96-a482-11e7-9e82-af1e1a11bca2.png)

**Comment by naelstrof** [comment]
Could you test if `maim -s`  retains the transparency? This is a new feature that has had little testing, I just want to make sure it works as I intend.

**Comment by AndrewOlsen** [comment]
Of course! Is what you wanted?

Full screenshot:
![full](https://user-images.githubusercontent.com/13722163/30994764-cb8fbd26-a483-11e7-87e2-ffb99c5d76ab.png)

The transparent frame:
![single](https://user-images.githubusercontent.com/13722163/30994771-e2ad1d8c-a483-11e7-8174-0bf33f2ea6fc.png)

**Comment by naelstrof** [comment]
Yes! This is what I wanted. Thank you. It definitely looks wrong, as if the transparent block is offset incorrectly. I'll have to take a look later. I'll leave this open until I figure out what's wrong.
